client/src/component/MilestoneTable.js

```diff
@@ -2,7 +2,7 @@ export function MilestoneTable() {
 
 	return (
 			<div>
-				<h1>hello world</h1>
+				<h1>Here we are waiting for Milestone Table</h1>
 			</div>
```

Comment:
thank you to aware us for waiting milestone table dear , but we can not add to another part of our teammate work , because it will cause conflict,i think it is better to remove this changes and push again

Reply:
I was confused by text and dislocation.

Reply:
removed changes